pkg/apis/stats/v1alpha1/types.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1alpha1
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+// +genclient
+// +genclient:nonNamespaced
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+type ClickHouseStats struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	Stat [][]string `json:"stat,omitempty"`
```

Comment:
Maybe we could make this as a structured data type instead of a matrix of string? Like
```golang
type Stats struct {
 Diskinfos []diskinfo `json:"diskinfos,omitempty"`
 ...
}
```
I just feel it is unclear what kind of Stat would be included when looking at this API.

Reply:
Updated. Please take a look to see if it is clear now?